src/services/File Management/controllers/filesControllers.js

```diff
@@ -0,0 +1,58 @@
+const { PrismaClient } = require("@prisma/client");
+const path = require("path");
+const fs = require("fs");
+const prisma = new PrismaClient();
+
+const storeFile = async (req, res) => {
+  try {
+    if (!req.file) {
+      return res.status(400).json({
+        status: "Fail",
+        message: "No File Uploaded.",
+      });
+    }
+    const Path = "uploads/" + req.file.filename;
+    const newFile = await prisma.file.create({
+      data: {
+        path: Path,
+      },
+    });
+    return res.status(200).json({
+      status: "Ok",
+      id: newFile.id,
+    });
+  } catch (error) {
+    console.error("Error uploading files:", error);
+    return res.status(500).json({
```

Comment:
Try using status codes like [this one](https://github.com/Mohab96/online-chess-game/blob/master/src/utils/statusCodes.js) as it's better than writing the status codes explicitly into the code like this.